spec/javascripts/indigenousLanguagesLayers_spec.js

```diff
@@ -0,0 +1,16 @@
+describe("Testing for indigenous Language layer", function() {
+
+  beforeEach(function () {
+      loadFixtures('indigenousLanguagesLayer.html');
```

Comment:
```suggestion
      loadFixtures('indigenousLanguagesLayers.html');
```

Reply:
Oh my, I cannot believe this was my error !!
Thanks a lot @jywarren !!!!!!!!